### PR TITLE
DOCS-1190: RELEASE.2024-04-18T19-09-19Z

### PR DESCRIPTION
**Pull request by ravindk89**
Partially addresses #1190 

No staged build unless someone requests it.

# Summary

After some thinking, we've leaned towards just making a note of the upgrade step and calling it done. We do not typically make extended discussion of backward-breaking changes at this time, and have no where to keep that sort of guidance.

Maybe in the future or as part of a larger re-arch we can consider where we would best keep that kind of guidance.

As far as the rest:

- We are not covering s3zip today, so that's a no-op. We should leave the ticket open in the event we have bandwidth to document the feature later
- No changes to Console that need documentation at this time
- Added a quick callout on the warning behavior to the Versioning page
- Metrics v3 is also still In progress, come back to this once the project is fully baked.

### CODE REVIEW

File: source/operations/install-deploy-manage/upgrade-minio-deployment.rst [hunk 1]
```diff
@@ -10,6 +10,10 @@ Upgrade a MinIO Deployment
    :local:
    :depth: 2
 
+.. important::
+
+   For deployments using :ref:`AD/LDAP <minio-ldap-config-settings>` for external authentication upgrading from a version older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```
Comment (feorlen) [inline source/operations/install-deploy-manage/upgrade-minio-deployment.rst:15]: possibly you intended this?
```suggestion
   For deployments using :ref:`AD/LDAP <minio-ldap-config-settings>` for external authentication and upgrading from a version older than :minio-release:`RELEASE.2024-03-30T09-41-56Z`: you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

Comment (feorlen) [reply source/operations/install-deploy-manage/upgrade-minio-deployment.rst:15]: also for the same text in upgrade tenant

Comment (ravindk89) [reply source/operations/install-deploy-manage/upgrade-minio-deployment.rst:15]:
```suggestion
   For deployments older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` running with :ref:`AD/LDAP <minio-ldap-config-settings>` enabled, you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

ThoughtS?

Comment (feorlen) [reply source/operations/install-deploy-manage/upgrade-minio-deployment.rst:15]: Yes!

File: source/operations/install-deploy-manage/upgrade-minio-tenant.rst [hunk 2]
```diff
@@ -10,6 +10,11 @@ Upgrade a MinIO Tenant
    :local:
    :depth: 1
 
+.. important::
+
+   For Tenants using :ref:`AD/LDAP <minio-ldap-config-settings>` for external authentication upgrading from a version older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before upgrading.
```
Comment (ravindk89) [inline source/operations/install-deploy-manage/upgrade-minio-tenant.rst:15]:
```suggestion
   For Tenants using a MinIO Image older than :minio-release:`RELEASE.2024-03-30T09-41-56Z` running with :ref:`AD/LDAP <minio-ldap-config-settings>` enabled, you **must** read through the release notes for :minio-release:`RELEASE.2024-04-18T19-09-19Z` before starting this procedure.
```

**Comment by ravindk89** [comment]
@feorlen back to you!

**Review by feorlen — APPROVED** [review]
👍🏻

**Pull request merged**